x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -605,25 +606,19 @@ const getQueryMappingForComparator = (comparator: Comparator) => {
   return queryMappings[comparator];
 };
 
-const getUngroupedResults = async (
-  query: object,
-  callCluster: LogThresholdAlertServices['callCluster']
-) => {
-  return decodeOrThrow(UngroupedSearchQueryResponseRT)(await callCluster('search', query));
+const getUngroupedResults = async (query: object, esClient: ElasticsearchClient) => {
+  return decodeOrThrow(UngroupedSearchQueryResponseRT)(await esClient.search(query));
```

Comment:
For the Log Threshold alert type this unfortunately breaks the executor:

```
[alerting][alerting][plugins][plugins] Executing Alert "bdf32c50-80d2-11eb-9f66-6dc90910fa35" has resulted in Error: Error: Failed to validate:
  in : undefined does not match expected type { _shards: { total: number, successful: number, skipped: number, failed: number }, timed_out: boolean, took: number }
  in 0: undefined does not match expected type { hits: { total: { value: number } } }
  in 1: undefined does not match expected type Partial<{ aggregations: { histogramBuckets: { buckets: Array<{ key: number, doc_count: number }> } } }>
```
We do an io-ts runtime check on the result, and the `UngroupedSearchQueryResponseRT` type doesn't match the new format (it's just missing the wrapping `body`). Passing `.body` to the `decodeOrThrow` instead will work.

(I'll add a note for adding a unit test, we have them for the query generation and results processing, but not that function that just calls ES.)